apps/browser/package.json

```diff
@@ -38,14 +38,17 @@
     "postcss": "8.4.31",
     "prettier-plugin-tailwindcss": "^0.5.4",
     "tailwindcss": "^3.3.3",
-    "typescript": "4.9.5"
+    "typescript": "4.9.5",
+    "supabase": "1.115.4"
   },
   "scripts": {
     "build": "next build",
     "dev": "next dev",
     "prebuild": "yarn build:scripts && yarn build:env",
     "build:scripts": "node scripts/build.scripts.js",
     "build:env": "node scripts/build.env.js",
+    "db:start": "supabase start",
```

Comment:
Should we add instructions on how to use this within the README?

Reply:
definitely! will do this tomorrow